app/src/main/java/com/kickstarter/libs/AttributionEvents.kt

```diff
@@ -28,6 +28,7 @@ class AttributionEvents(
     fun trackProjectPageViewed(projectData: ProjectData) {
         val props: HashMap<String, Any> = hashMapOf(ContextPropertyKeyName.CONTEXT_PAGE.contextName to EventContextValues.ContextPageName.PROJECT.contextName)
         props["context_page_url"] = projectData.fullDeeplink()?.toString() ?: ""
+        props["session_device_type"] = "phone" // Remove when https://kickstarter.atlassian.net/browse/MBL-1275 is complete
```

Comment:
Manually overwrite the value of "session_device_type" because our Android User Agent is currently not supplying the backend with correct system information and so the backend will default to "desktop" as the value unless we overwrite it.

Reply:
i'd recommend making this a todo just so it can be found that way if necessary